R/zzz.R

```diff
@@ -0,0 +1,48 @@
+if(getRversion() >= "2.15.1") utils::globalVariables(
+  c(
+    # Oracle SQL
+    "INSTR",
+    "LENGTH",
+    "REGEXP_LIKE",
+    "REGEXP_REPLACE",
+    "REPLACE",
+    "SUBSTR",
+    "UTL_MATCH.JARO_WINKLER",
+
+    # AddressBase table columns
+    "DPA_SINGLE_LINE_ADDRESS",
+    "DEPARTMENT_NAME",
+    "RM_ORGANISATION_NAME",
+    "SUB_BUILDING_NAME",
+    "BUILDING_NAME",
+    "BUILDING_NUMBER",
+    "PO_BOX_NUMBER",
+    "DEP_THOROUGHFARE",
+    "THOROUGHFARE",
+    "DOU_DEP_LOCALITY",
+    "DEP_LOCALITY",
+    "POST_TOWN",
+    "POSTCODE",
+    "GEO_SINGLE_LINE_ADDRESS",
+    "LA_ORGANISATION",
+    "SAO_TEXT",
+    "SAO_START_NUMBER",
+    "SAO_START_SUFFIX",
+    "SAO_DASH",
+    "SAO_END_NUMBER",
+    "SAO_END_SUFFIX",
+    "PAO_TEXT",
+    "PAO_START_NUMBER",
+    "PAO_START_SUFFIX",
+    "PAO_DASH",
+    "PAO_END_NUMBER",
+    "PAO_END_SUFFIX",
+    "STREET_DESCRIPTION",
+    "LOCALITY",
+    "TOWN_NAME",
+    "POSTCODE_LOCATOR",
+
+    # Created columns that do not work with .data pronoun
+    "SCORE"
```

Comment:
`dplyr::slice_max ` does not seem to handle `.data`. Would expect it to, as the documentation says it uses data-masking.

Hence why https://github.com/nhsbsa-data-analytics/addressMatchR/pull/38/files#diff-46de35c1814e363c6cb14e19349e8a03ae1311fbba3f26349fa1a5e95edc3736R171 uses bare `SCORE`.